test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -345,6 +346,39 @@ def test_non_clifford_interleaved_element(self):
             lengths=lengths,
         )
 
+    def test_interleaving_delay(self):
```

Comment:
I think both these test requires qiskit-terra 0.20.1 to not error

Reply:
Good point. All test passed in CI because it simply builds env with the latest version?

Reply:
Yeah it will pull the latest stable in CI, but if someone updates an existing qiskit-experiment install from pip it wont automatically update terra unless we change the requirements.

Reply:
Right, this should be merged after the requirements change.

Reply:
Updated the minimum requirements in c5e66cd